### DISCUSSION TITLE
Unabel to save configuration

### DISCUSSION THREAD

**Discussion by Sceptorrh** [cgiesche/streamdeck-homeassistant#264] [Help and Support]
Hello,

I am unable to safe the configuration. Am I doing something wrong. I installed the plugin trough the marketplace,

Created a long lived token in HA

connection through: wss://homeassistant.mydomain.nl/api/websocket

Connection is successful and I can see my entities, but pressing Save Configuration does nothing.

**Comment by cgiesche** [comment]
Hi,

someone had the exact same issue a week or two ago. The issue did somehow disappear. Maybe try to restart the stream deck application (or the PC). I have no idea what causes this. Seems like this happens only on "first time installations".

Sent from Nine

________________________________
Von: Bradley ***@***.***>
Gesendet: Donnerstag, 21. März 2024 19:44
An: cgiesche/streamdeck-homeassistant
Cc: Subscribed
Betreff: [cgiesche/streamdeck-homeassistant] Unabel to save configuration (Discussion #264) 

Hello,
I am unable to safe the configuration. Am I doing something wrong. I installed the plugin trough the marketplace,
Created a long lived token in HA
connection through: wss://homeassistant.mydomain.nl/api/websocket
Connection is successful and I can see my entities, but pressing Save Configuration does nothing.

**Reply by Sceptorrh** [reply]
Hey, Thanks for the quick reply. 

I tried reinstalling the Elgato app (including removing the appdata).
I am also able to connect locally but I am sadly still unable to save the settings.

Please let me know If I can help to troubleshoot!

Running homeassistant version:
- Core 2024.3.1
- Supervisor 2024.03.0
- Operating System 12.1
- Frontend 20240307.0

Elgato software
 - 6.5.2 (19936)

Homeassistant plugin
 - 2.4.5

Windows
 - Windows 11 Pro (23H2)

Streamdeck hardware
- Streamdeck XL

The steps I took where
 - Remove Elgato from windows apps
 - Remove Elgato folder from AppData
 - Remove Elgato folder from ProgramFiles
 - Reinstall Elgato application
 - Reinstall HomeAssistant plugin trough Marketplace
 - Connect to Homeassistant using local url & long lived access token
 - Set entity settings (Lights, toggle)
 - And then trying to save. But that doesn't seem to work for me

**Reply by Sceptorrh** [reply]
Just tested on a different pc (My laptop)
Installed fresh version of Elgato software and the plugin. Work just fine...

Seems my installation is broken somehow
